rln/src/poseidon_tree.rs

```diff
@@ -105,5 +242,270 @@ mod test {
             "   - Optimal Merkle Tree: {:?}",
             Duration::from_nanos((upd_time_opt / sample_size).try_into().unwrap())
         );
+
+        println!(
+            "   - Pmtree-HashMap Merkle Tree: {:?}",
+            Duration::from_nanos((upd_time_pm_memory / sample_size).try_into().unwrap())
+        );
+
+        println!(
+            "   - Pmtree-Sled Merkle Tree: {:?}",
+            Duration::from_nanos((upd_time_pm_sled / sample_size).try_into().unwrap())
+        );
+    }
```

Comment:
Maybe we can benchmark using the in-built lib - https://doc.rust-lang.org/1.16.0/book/benchmark-tests.html? (the lib is unstable, so up to you)

Reply:
Sounds good. Tracking here - https://github.com/vacp2p/zerokit/issues/89